app/controllers/issues_controller.rb

```diff
@@ -39,6 +44,10 @@ def index
       last_updated_by = params[:last_updated_by].to_s == "nil" ? nil : params[:last_updated_by].to_i
       @issues = @issues.where(:updated_by => last_updated_by)
     end
+
+    @issues, @newer_issues_id, @older_issues_id = get_page_items(@issues, :limit => @params[:limit])
+
+    render :partial => "page" if turbo_frame_request_id == "pagination" || turbo_frame_request_id == "search"
```

Comment:
I think the second part of this condition is wrong (in fact it's not clear to be the `search` frame is needed at all) and instead the search form should have a `data-turbo-frame="pagination"` attribute to target loading of the pagination frame.

Reply:
It was written to update only the list part of the page when the search button was clicked. But it may be out of scope of this PR (as it introduces new action other than pagination). I'll remove and it may be a separate PR when we will finish pagination functionality.

Reply:
Right, but you can do that by telling the search form to target the `pagination` frame, which is all that is actually being updated given that is what you render.

If you thought the search form needed to be updated as well then the above should be rendering a larger section for `search` but I don't think that does need to be updated does, only the results (which are in the `pagination` frame) need to be updated?